packages/utils/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@respond-framework/utils",
+  "version": "0.1.0",
```

Comment:
lets also set up commitizen and semantic-release. If we can disable npm publishing from the automation - we can at least get a changelog going. Ive set it up once or twice but your eyes on things generally make it smoother. So take a peek at me idea and ill help build it in to all the separate packages

Reply:
made this to track it: https://github.com/respond-framework/rudy/issues/11